src/drv_vrtek/vrtek.c

```diff
@@ -0,0 +1,552 @@
+// Copyright 2019, Mark Nelson.
+// SPDX-License-Identifier: BSL-1.0
+/*
+ * OpenHMD - Free and Open Source API and drivers for immersive technology.
+ */
+
+/* VR-Tek Driver */
+
+#include <stdlib.h>
+#include <hidapi.h>
+#include <string.h>
+#include <stdio.h>
+#include <assert.h>
+
+#include "vrtek.h"
+#include "../hid.h"
+
+/* VR-Tek reuses the Oculus Vendor ID */
+#define OCULUS_VR_INC_ID       0x2833
+#define VRTEK_WVR_HMD          0x0001
+
+typedef enum {
```

Comment:
In general we tend to place enum's in the header file, could you move these please?

Reply:
Ahh, I didn't pick up on that convention, sorry. I just pushed a commit that does just that.